packages/pointer_interceptor/pointer_interceptor/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.9.4
+* Adds iOS implementation
```

Comment:
Since it is a breaking change, version should be 0.10.0
And there should be a space between the version and message, and a "." at the end of the message.